homeassistant/components/ssdp/__init__.py

```diff
@@ -559,3 +594,171 @@ def _udn_from_usn(usn: str | None) -> str | None:
     if usn.startswith("uuid:"):
         return usn.split("::")[0]
     return None
+
+
+class HassUpnpServiceDevice(UpnpServerDevice):
+    """Hass Device."""
+
+    DEVICE_DEFINITION = DeviceInfo(
+        device_type="urn:home-assistant.io:device:HomeAssistant:1",
+        friendly_name="Home Assistant",
```

Comment:
Should this be the location name? `hass.config.location_name`

Reply:
I'm uncertain. When I use that the icon has the name "Home". I think it might not be clear that this is their home assistant when a new user sees this icon/name.

![image](https://user-images.githubusercontent.com/724259/195920914-80a65006-0fcc-40d3-8855-ebbf87ebe321.png)


How about `f"Home Assistant - {hass.config.location_name}"`? I.e., "Home Assistant - Home". Then, it is clear what it is and if somebody has multiple instances one can still see which one is which.

Reply:
Maybe "Home (Home Assistant)" ?

Reply:
![image](https://user-images.githubusercontent.com/724259/195981850-d20dcb85-432e-4f20-a516-9eb95e67034c.png)